secrets/aws-secret-manager/aws_secret_access_key copy

```diff
@@ -0,0 +1 @@
+quOEpzR8mh/yp2Nfjhwi7YBamTZVEaSJbtmTblIQ
```

Comment:
I see secrets commited here:
Some seem copies so I guess it is not intentional.
Other seem real secrets that were in the local repository. This is a security risk. Have the secrets been rotated?

Reply:
These are dummy secrets. Actually, this file should be ignored - I wonder why it is there.

Reply:
Are you sure those are dummy secrets?

Reply:
Yes.